membersrvc/ca/ca_test.yaml

```diff
@@ -339,3 +339,8 @@ pki:
                             file: testdata/server1.key
 
             devops-address: 0.0.0.0:30303
+
+        ca:
+            subject:
+                organization: Hyperledger
+                country: US
```

Comment:
It's valid YAML. I think GitHub is just letting you know that it's the end of the file. It's often considered good style to put a new line at the end of a file. My editor does it automatically.